### ISSUE TITLE
Nonce is not added to CSP header in rails 5

### ISSUE THREAD

**Issue by afcapel** [twitter/secureheaders#285]
## Steps to reproduce

1. On a brand new rails 5.0.0.1 project add the `secure_headers` gem.
2. Create an initializer including `SecureHeaders::Configuration.default` 
3. Create an action and view including:

```
<h1>Hello</h1>

<%= nonced_javascript_tag do %>
  alert("world!");
<% end %>
```

### Expected outcome

The script tag is rendered including a `nonce` attribute and a `nonce-X` origin is added to the `Content-Security-Policy` header.

### Actual outcome

The script tag correctly includes a nonce attribute, but there's no nonce origin in the `Content-Security-Policy` HTTP header, so the browser refuses to execute the inline script.

**Comment by oreoshake** [comment]
Ouch. I'll take a look at this shortly.

**Comment by oreoshake** [comment]
@afcapel can you provide your `SecureHeaders::Configuration.default` block? I can only reproduce this if I do not have a `script-src` value set. If that's the case, this bug isn't as awful as I had thought :smile:

While it's always better to supply the `script-src` value rather than falling back to `default-src`, the library is supposed to handle this gracefully and copy over the `default-src` and include the nonce.

**Comment by oreoshake** [comment]
Fix released in 3.4.1
